src/MassTransit/Configuration/Configuration/ReceiverConfiguration.cs

```diff
@@ -44,6 +44,10 @@ ConnectHandle IReceiveEndpointObserverConnector.ConnectReceiveEndpointObserver(I
             return _configuration.ConnectReceiveEndpointObserver(observer);
         }
 
+        public void AddDependent(IReceiveEndpointObserverConnector connector)
```

Comment:
Wouldn't this be IReceiveEndpointDependentConnector ?

Reply:
I guess this one is `IReceiveEndpointDependencyConnector` which doesn't make sense, I only wanna connect Observer then